src/components/AppContentListItem/AppContentListItem.vue

```diff
@@ -0,0 +1,366 @@
+			console.debug('menuopen', newValue)
+			// A click outside both the menu and the root element hides the actions again
+			if (!newValue && !this.hovered) {
+				this.displayActions = false
+			}
+		},
+	},
+
+	methods: {
+
+		// forward click event
+		onClick(event) {
+			this.$emit('click', event)
+		},
+
+		handleHover() {
+			this.showActions()
+		},
+
+		handleFocus() {
+			this.focused = true
+			this.showActions()
+		},
+
+		handleBlur(e) {
+			this.focused = false
+		},
+
+		showActions() {
+			if (this.hasActions) {
+				this.displayActions = true
+			}
+		},
+
+		handleMouseleave() {
+			if (!this.menuOpen) {
+				this.displayActions = false
+			}
+		},
+
+		handleTab(e) {
+			if (this.focused && this.hasActions) {
+				console.debug('preventdefault')
+				e.preventDefault()
+				this.$refs.actions.$refs.menuButton.focus()
+				this.focused = false
+			} else {
+				this.displayActions = false
+				this.$refs.actions.$refs.menuButton.blur()
+			}
+		},
+
+		handleActionsUpdateOpen(e) {
+			this.menuOpen = e
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+
+.acli_wrapper{
+	position: relative;
+	width: 100%;
+	.actions {
+		position: absolute;
+		top: 4px;
+		right: 12px;
+	}
+}
+
+// AppContentListItem
+.acli {
+	position: relative;
+	display: flex;
+	align-items: center;
+	flex: 0 0 auto;
+	justify-content: flex-start;
+	padding: 2px 2px 2px 8px;
+	height: 64px;
+	border-radius: 16px;
+	margin: 2px 0;
+	width: 100%;
+	cursor: pointer;
+	transition: background-color 100ms ease-in-out;
+	&:hover,
+	&:focus  {
+		background-color: var(--color-background-hover);
+	}
+	&.active,
+	&:active,
+	&:active ~ .app-navigation-entry__utils {
+		background-color: var(--color-primary-light);
+	}
+
+	&-content {
+		display: flex;
+		flex: 1 1 auto;
+		justify-content: space-between;
+		padding: 0 8px;
+
+		&__main {
+			flex: 1 1 auto;
+			flex-direction: column;
+			width: 0;
+			margin: auto 0;
+		}
+
+		&__line-one {
+			display: flex;
+			align-items: center;
+			justify-content: space-between;
+			white-space: nowrap;
+			margin: 0 auto;
+
+			&__title {
+				overflow: hidden;
+				flex-grow: 1;
+				cursor: pointer;
+				text-overflow: ellipsis;
+				color: var(--color-main-text);
+
+			}
+		}
+
+		&__line-two {
+			display: flex;
+			align-items: flex-start;
+			justify-content: space-between;
+			white-space: nowrap;
+
+			&__subtitle {
+				overflow: hidden;
+				flex-grow: 1;
+				padding-right: 4px;
+				cursor: pointer;
+				white-space: nowrap;
+				text-overflow: ellipsis;
+				color: var(--color-text-lighter);
+			}
+		}
+		&__actions {
+			flex: 0 0 auto;
+			align-self: center;
+			justify-content: center;
+
+		}
+	}
+}
+
+// Counter
+::v-deep .counter {
+	font-size: 12px;
+	/*
+	 * Always add the bubble
+	 */
+	padding: 4px 6px !important;
+	border-radius: 10px;
+
+	&:not(.app-navigation-entry__counter--highlighted) {
+		background-color: var(--color-background-darker);
+	}
+
+	span {
+		padding: 2px 6px;
+	}
+}
```

Comment:
It feels a bit weird, no?
To have custom css for an appnavigationCounter within an appCOntentListItem ?

Reply:
Yeah, you're right. Should we add a toggle for this style in AppNavigationCounter?

Reply:
I don't know, can't we just change the style of the counter?

Reply:
I thought that that counter without bubble was needed somewhere, but sure!

Reply:
Pr here @skjnldsv https://github.com/nextcloud/nextcloud-vue/pull/1769